corio.py

```diff
@@ -136,7 +136,7 @@ def main(options):
     collect_resource_utilisation(action="start")
     if jira_flg:
         jira_obj = JiraApp()
-        tests_details = jira_obj.get_all_tests_details_from_tp(options.test_plan)
+        tests_details = jira_obj.get_all_tests_details_from_tp(options.test_plan, reset_status=True)
```

Comment:
If we have this as default Reset, and user submit execution on the same TestPlan by mistake, results will be overwritten.
Instead of making it default, it is better to take this as input and keep default as False, so that user will understand what is happening and why is it happening, since user will read doc to understand parameter.